charts/refinery/templates/deployment-redis.yaml

```
@@ -16,15 +16,27 @@ spec:
   {{- include "refinery.redis.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
+      {{- /*
+        FIXME .Values.podAnnotations should not be used at all here.
+        Keeping it for backward compatibiltiy but should eventually be removed.
+      */}}
```

```suggestion
      {{- /*
       .Values.podAnnotations is used here to maintain backwards compatibility.
       It should be removed eventually.
      */}}
```

I _think_ that comment won't get rendered.